packages/ui-extensions/docs/surfaces/point-of-sale/mdxExamples/discount-example/modal.ts

```diff
@@ -7,6 +7,7 @@ import {
 } from '@shopify/ui-extensions/point-of-sale';
 
 export default extension('pos.home.modal.render', (root, api) => {
+  // [START modal.define-onpress]
```

Comment:
I'm assuming these "comments" get ignored when referenced on the MDX?

Reply:
Correct! These comments are used to actually link each tutorial step.